content/en/blog/_posts/2023-09-20-kubeadm-use-etcd-learner-mode.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: blog
+title: 'Kubeadm: Use ETCD Learner to join a control plane node safely'
```

Comment:
```suggestion
title: 'Kubeadm: Use Etcd Learner to Join a Control Plane Node Safely'
```

Reply:
Updated.

`title: 'Kubeadm: Use etcd Learner to Join a Control Plane Node Safely'`